operators/kong-gateway-operator/1.0.0-rc.0/manifests/kong-gateway-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1163 @@
+          - get
+          - patch
+          - update
+        - apiGroups:
+          - configuration.konghq.com
+          resources:
+          - tcpingresses
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - configuration.konghq.com
+          resources:
+          - tcpingresses/status
+          verbs:
+          - get
+          - patch
+          - update
+        - apiGroups:
+          - configuration.konghq.com
+          resources:
+          - udpingresses
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - configuration.konghq.com
+          resources:
+          - udpingresses/status
+          verbs:
+          - get
+          - patch
+          - update
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - patch
+        - apiGroups:
+          - ""
+          resources:
+          - namespaces
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - nodes
+          verbs:
+          - list
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - secrets
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - serviceaccounts
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - serviceaccounts/status
+          verbs:
+          - get
+        - apiGroups:
+          - ""
+          resources:
+          - services
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - services/status
+          verbs:
+          - get
+          - patch
+          - update
+        - apiGroups:
+          - discovery.k8s.io
+          resources:
+          - endpointslices
+          verbs:
+          - get
+          - list
+          - watch
```

Comment:
@mporrato @Allda Can you take a look at this error message? https://github.com/k8s-operatorhub/community-operators/actions/runs/6276653150/job/17046897650#step:6:1370

```
      {
          "passed": true,
          "outputs": [
              {
                  "type": "warning",
                  "message": "Warning: Value kong-gateway-operator.v1.0.0-rc.0: this bundle is using APIs which were deprecated and removed in v1.25. More info: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-25. Migrate the API(s) for endpointslices: ([\"ClusterServiceVersion.Spec.InstallStrategy.StrategySpec.ClusterPermissions[0].Rules[32]\"])"
              },
              {
                  "type": "warning",
                  "message": "Warning: Value kong-gateway-operator.v1.0.0-rc.0: this bundle is using APIs which were deprecated and removed in v1.25. More info: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-25. Migrate the API(s) for endpointslices: ([\"ClusterServiceVersion.Spec.InstallStrategy.StrategySpec.ClusterPermissions[0].Rules[32]\"])"
              }
          ]
      }
```

our bundle uses `EndpointSlices` which v1beta1 was removed in 1.25 https://kubernetes.io/docs/reference/using-api/deprecation-guide/#endpointslice-v125 but it's still served in v1.

Our bundle uses that but is not specifying the version here. This shouldn't be an issue though, right?

Reply:
Perhaps the version of operator-sdk is relevant here?

https://github.com/k8s-operatorhub/community-operators/pull/3276/files#diff-395e3b4f7fa453efeb7497ffa9774a429877b9377237c9dba67405c093217d56R10

```
LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.23.0
```

Reply:
This is just a warning, and is not the root cause of the error. The root cause is that at like 1050, there isn't a registry being specified for the controller image.

Reply:
Ok, thanks! Closing this PR as an automation will resubmit.